mitosheet/mitosheet/public/v2/excel_utils.py

```diff
@@ -132,7 +146,7 @@ def get_read_excel_params_from_range(range: str) -> Tuple[int, int, str]:
     return start_row_index, nrows, usecols
 
 
-def convert_csv_file_to_xlsx_file(csv_path: str, sheet_name: str) -> str:
+def convert_csv_file_to_xlsx_file(csv_path: str, sheet_name: Union[str, int]) -> str:
```

Comment:
We should add a comment here too that these parameters can not change order for backwards compatibility. It would be nice to test that in fact.

Reply:
Added test.